src/externals/mct/mct/m_GlobalSegMap.F90

```diff
@@ -1764,22 +1761,25 @@ subroutine active_pes_(GSMap, n_active, pe_list)
   character(len=*),parameter :: myname_=myname//'::active_pes_'
 
   integer :: count, i, n, ngseg, ierr
-  logical :: new
-  integer, dimension(:), allocatable :: temp_list
-  integer, dimension(:), allocatable :: perm
+  integer :: max_activepe, p
```

Comment:
Of course my name is on that because it was a PR bringing in a new version of MCT as a CIME subtree. That doesn't mean I made the changes to MCT (or to mpi-serial which was brought into MCT as a new subtree). You need to look in those repositories for the true source of the changes. Also, note that you merged the PRs of mpi-serial into MCT.